sceptre/scipool/config/prod/sc-tag-options.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
 sceptre_user_data:
   Departments: !file sc-tag-options/internal/Departments.json
   Projects: !file sc-tag-options/internal/Projects.json
-  CostCenters: !file https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.6/tags/SageFinancialProgramCodes.json
+  CostCenters: !request https://finops-api.sageit.org/tags?limit=25
```

Comment:
I want to move this to `https://mips-api.finops.sageit.org/tags?limit=25` so that I can use `https://cost-rules.finops.sageit.org/` for the cost-rules lambda: https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/775

Reply:
ok, can we go ahead with this PR and update this URL after you change the endpoint?